src/ds5/ds5-device.cpp

```diff
@@ -699,6 +722,11 @@ namespace librealsense
         auto& raw_sensor = get_raw_depth_sensor();
         auto pid = group.uvc_devices.front().pid;
 
+        _color_calib_table_raw = [this]()
+        {
+            return get_raw_calibration_table(rgb_calibration_id);
+        };
+
```

Comment:
I think it may interfere with D420/430 SKUs and the internal call may throw.
For review